src/test/groovy/org/prebid/server/functional/tests/AliasSpec.groovy

```diff
@@ -29,12 +31,16 @@ class AliasSpec extends BaseSpec {
 
         and: "Resolved request should contain aliases as in request"
         assert responseDebug.resolvedRequest.ext.prebid.aliases == bidRequest.ext.prebid.aliases
+
+        and: "Bidder request should contain request per-alies"
+        def bidderRequests = bidder.getBidderRequests(bidRequest.id)
+        assert bidderRequests.size() == 2
```

Comment:
Please add a parameter to aliased bidder config and check that it's properly passed downstream. Plus check that it's not present in the generic bidder call. You can create a separate test for that if you want. Call it `PBS aliased bidder config should be independentl from parent`